index.html

```diff
@@ -0,0 +1,93 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+
+    <main class="main-page">
+      
+        <div class="title">
+          <h1>Welcome to Microverse Remote Courses</h1>
+          <p class="description"> Thank you for taking the time to help us improve the platform</p>
+        </div>
```

Comment:
In [index.html](https://github.com/joel-kalema/Portfolio/pull/2#discussion_r803633850):

[OPTIONAL] You can improve it even more since it is a `header` 🤔, in this particular case, it is the first section on this page. So why don't we try to use a <header />? 😃 . Sample code:

<header>

  <img src="icon_path" alt="mob menu icon" />

              <div class="title">
                    <h1>Welcome to Microverse Remote Courses</h1>
                    <p class="description"> Thank you for taking the time to help us improve the platform</p>
               </div>

</header>

Reply:
[REQUIRED] Great work on sounding polite as it was requested by the previous code reviewer :+1: However, this comment lacks a clear and solid reason for the requested change. For example, we are saying:

_"instead of using div tags to group elements, we can use header, nav, or section tags depending on what specific type of group of elements we are trying to make"_

However, 
- we are not saying how this change can benefit the codebase; 
- we are not saying why making this change is worth the time and effort spent on the amendment;

That is very important, because, let's take ourselves (Microverse students)  as an example - Every time we submit for CR and if we are required to make some change to your project, the first thing that comes to our mind as a developer is a question `Why?`  and as a code reviewer, answering that question is on of our responsibilities. Please take an example of `good comment `and `bad comment` below:

**This is a well-explained request**: _Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible_

**This request lacks a "why"**: _Why don't you use an article tag here instead of a div?_

- [ ] Please kindly amend this comment as based on the above remarks
- [ ] [OPTIONAL] Please kindly make sure you are suggesting the correct option otherwise the developer might get lost. In your sample code, I can see that you put the `<div/>` element inside your `<header/>` and inside you have `<h2/>` and `<p/>`. Please note kindly note that the `<header/>` element is usually used for holding logos, toolbar, navigations, menu buttons etc:

```HTML
 <header>
    <hgroup>
      <h1>...</h1>
      <h2>.....</h2>
    </hgroup>
    <nav>
      <ul>
        <li><a href="#">...</a></li>
        <li><a href="#">...</a></li>
        <li><a href="#">...</a></li>
        <li><a href="#">...</a></li>
        <li><a href="#">...</a></li>
      </ul>
    </nav>
  </header>
```